user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -20,9 +20,16 @@ A Catalog item is identified by a unique ID (GUID), which you will need to provi
 
 The register API call allows you to create or update a Catalog item. To add a version after you have successfully registered an item, see [Registering a new version with the API](#registering-a-new-version-with-the-api).
 
-### URL
+### API Definition
 
-`https://api.dataminer.services/api/key-catalog/v1-0/catalog/register`
+For a complete definition of the API, go to [Key Catalog API Swagger](https://catalogapi-prod.cca-prod.aks.westeurope.dataminer.services/swagger/index.html?urls.primaryName=Key+Catalog+API+v1.0).
+
+This page also provides an easy way to try out the call: Expand the "catalog/register" item, and click the *Try it out* button.
+
+> [!IMPORTANT]
+> The *Try it out* button allows you to execute the register call on the production Catalog.
```

Comment:
indeed. you will perform an action on the actual Catalog data

Reply:
Doesn't this make the "Try it out" title of the button a little misleading? Users might get the impression that they're just doing a harmless test, but they're actually making changes in the Catalog.

Reply:
I can see people can have that impression but we don't have full control on what is being shown on that page so unfortunately we cant change the text. Best alternative is perhaps improving that warning message? I do believe it will be clear for people familiar with swagger/apis